fossunited/api/proposal.py

```diff
@@ -0,0 +1,41 @@
+import frappe
+
+from fossunited.fossunited.utils import get_doc_likes
+
+
+@frappe.whitelist(allow_guest=True)
+def get_event_proposals(
+    event: str,
+) -> list:
+    """
+    Get all the proposal submissions for the given event.
+
+    Adds number of likes, and redacts speaker name based on is_anonymise condition
+
+    Args:
+        event (str): The id of the event
+
+    Returns:
+        list: proposals of an event
+    """
+
+    fields = [
+        "name",
+        "route",
+        "talk_title",
+        "session_type",
+        "full_name",
+        "status",
+    ]
+
+    proposals = frappe.get_all("FOSS Event CFP Submission", filters={"event": event}, fields=fields, page_length=99999, order_by="talk_title")
+
```

Comment:
nit: shouldn't we be ordering stuff by creation?

Reply:
by talk proposals would be easier to find in a way - used to do it for previous years'